.pre-commit-config.yaml

```diff
@@ -9,24 +9,31 @@ repos:
         files: \.(py|sh|rst|yml|yaml)$
     -   id: trailing-whitespace
         files: \.(py|sh|rst|yml|yaml)$
+-   repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.270
+    hooks:
+    -   id: ruff
+        args: [--fix, --exit-non-zero-on-fix]
 -   repo: https://github.com/adamchainz/django-upgrade
     rev: 1.13.0
     hooks:
     -   id: django-upgrade
         args: [--target-version, "4.2"]
--   repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-    -   id: isort
+# Replaced by ruff:
```

Comment:
isort gone? 😢

Reply:
Added back in e0151cfbb6da5044a4b3403e75624a4c22786877